### ISSUE TITLE
BUG - sev-2 - All - Availability Framework: AF banner doesn't scroll

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#7259]
## What happened?

The [availability framework banner doesn't scroll with the rest of the page content](https://drive.google.com/file/d/1Z7czqHRPhKM74vIVq0oWGBsefvyzZpTJ/view?usp=drive_link) - it's locked in place, and the non-AF stuff scrolls within the remaining available viewing space on the device.

This is ESPECIALLY problematic in landscape mode or with text resized larger, because the AF banner can take up the entire viewable portion of the screen -- so when it doesn't scroll, you 1) can't see the entire AF banner (so could be missing information, or the ability to quickly update the app with the button we provide) and 2) can't use any of the screen (in use case 3 workflows).

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by Sparowhawk** [comment]
this is related to the VCL banner, if you move the wrapper into the scrollview just surrounding the children component it would work as intended

**Comment by TKDickson** [comment]
Testing for #7255 and this ticket in a single PR

Need to check scrolling (mix of use case 2 and 3, since they're the same component styled differently) on every template:
- [ ] Category landing
- [ ] Child/feature
- [ ] Task/subtask
- [ ] Large panel

Check VCL & AF banner hierarchical display, on as many different templates as VCL appears
- [ ] Category landing (homepage)
- [ ] Child/feature (health)
- [ ] Task/subtask (probably somewhere in messaging, will replace with actual screen if I find one)
- [ ] Large panel (probably somewhere in messaging, will replace with actual screen if I find one)
 
Also will be finally looping around to text/display resize testing, again with each of the templates.
- [ ] Category landing
- [ ] Child/feature
- [ ] Task/subtask
- [ ] Large panel

**Comment by TKDickson** [comment]
I've found two issues with this build with the selected cases above:
- Status label does not open *at all* (doesn't matter if I have AF enabled or not)
- Bottom action buttons still appear even if content on the page is denied, see screenshot of refill modal screen

<img width="366" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTR0QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e43fbd6944bea3d0dfe613a4381add986f450c37/image.png" alt="image.png" />

**Comment by TKDickson** [comment]
New cases for the fix, based on the comment above + a quick call with Binny and Theo.

First fix
- [ ] Can open Veteran Status (at all!)
- [ ] Can open Rx status label (at all!)
- [ ] AF banner on Veteran Status looks good, scrolls, positioned correctly, etc
- [ ] AF banner on Rx status label looks good, scrolls, positioned correctly, etc
----
Second fix
- [ ] Use Case 1 on a 'broken use case 1' screen, ideally one with a template that's not Home/Category -- nothing unexpectedly weird happens (still shows broken use case 1 UI, but doesn't have two alerts for instance)
- [ ] Use Case 2 on a subtask/task template screen with a bottom button (Refill modal, like the original found bug)
- [ ] Use Case 3 on a subtask/task template screen with a bottom button (Refill modal) <-- _this shouldn't at all be impacted by the change, just regression-testing_
- [ ] Use Case 2 on a child/feature template screen to make sure only one alert appears (claims history)
- [ ] Use Case 2 on a large panel template screen to make sure only one alert appears  (what to know before sharing gender ID)

**Comment by TKDickson** [comment]
The previously-broken UC1 screens that were described and not fixed in ticket #7282 (for instance - the home and category screens, VCL) no longer display the broken UC1 behavior in that ticket. Now if you set up a UC1 alert for one of those screens, nothing shows up - no UC1 popup, and no mixed-UI-broken-UC1 alert banner. This is probably a step in the right direction, actually -- but will confirm with UX/Product, and document on our ['weird stuff about AF' page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/features/Remote-Disaster-Mitigation/Framework_notes_DRAFT.md), before signing off on this ticket.

**Comment by TKDickson** [comment]
JK, that change predates this ticket. Documented the change in the weird stuff page, and this change is good to merge to develop when the rest of AF does!
